code/datums/uplink/devices and tools.dm

```diff
@@ -10,6 +10,11 @@
 	path = /obj/item/storage/toolbox/syndicate
 	desc = "Danger. Very robust. Filled with advanced tools."
 
+/datum/uplink_item/item/tools/gentlemen_kit
+	name = "Gentlemen's Kit"
```

Comment:
`Gentlemen` is plural, `gentleman` is singular. Pretty sure you meant second one?

Reply:
my mistake, should be gentelman